src/messagingapplication.cpp

```diff
@@ -265,6 +268,9 @@ void MessagingApplication::parseArgument(const QString &arg)
     Q_FOREACH(const Pair &item, query.queryItems(QUrl::FullyDecoded)) {
         if (item.first == "text") {
             properties["text"] = item.second;
+        }
+          if (item.first == "text") {
```

Comment:
why duplicating that condition ?

Reply:
whops, didnt finish that. was going to add 'body'  for when thr url  has ?body= is that the right place?

Reply:
yes, i think it is the place, but i've just tested, the "text" query param is not handled in the message, so some works need to be done in Messages.qml